### ISSUE TITLE
Opening gitup error - failed to read descriptor: illegal byte sequence

### ISSUE THREAD

**Issue by chmac** [git-up/GitUp#702]
I'm seeing this error when I try to start gitup in one repository:

<img width="404" alt="image" src="https://user-images.githubusercontent.com/690997/92948184-47ace000-f459-11ea-835e-5243f7289aee.png">

Any suggestions on how to investigate? I tried `brew reinstall gitup` to no avail. It's only in 1 repo. I guess this repo was open in gitup recently while my mac crashed. Not sure if that's related somehow.

**Comment by lucasderraugh** [comment]
Is the non-working repo publicly available to test with?

**Comment by chmac** [comment]
@lucasderraugh Nope, the issue is with my filesystem I believe. Something is corrupted as I can't `git fetch` in that repo. Not sure if there's much to do about it, I guess the error message could be more verbose, but `git` itself also spits out a pretty confusing error also.
